src/selectors/entities/groups.js

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {createSelector} from 'reselect';
+
+export function getAllGroups(state) {
+    return state.entities.groups.groups;
+}
+
+export const getGroup = createSelector(
```

Comment:
Most of these selectors should be plain functions since they have arguments and they're mostly returning objects straight from the store.

Reply:
Fixed.